content/content/oop_macro.md

```diff
@@ -43,7 +43,7 @@ To get that nice notation, we can use a macro:
 ```nimrod
 import macros
 
-macro class*(head: expr, body: stmt): stmt {.immediate.} =
+macro class*(head, body): untyped =
```

Comment:
`head, body` should be explcitily annotated as untyped. ie, `head, body: untyped`